Sources/Tracing/Docs.docc/Guides/ImplementATracer.md

```diff
@@ -163,7 +163,7 @@ extension Baggage {
   }
 }
 
-var context = Baggage.topLevel(logger: ...)
-context.baggage.traceID = "4bf92f3577b34da6a3ce929d0e0e4736"
-print(context.baggage.traceID ?? "new trace id")
+var context = ServiceContext.topLevel(logger: ...)
+context.context.traceID = "4bf92f3577b34da6a3ce929d0e0e4736"
```

Comment:
```suggestion
context.traceID = "4bf92f3577b34da6a3ce929d0e0e4736"
```
The extension is on `ServiceContext` so we shouldn't need the nested `context` here.